src/components/atoms/Input/Input.tsx

```diff
@@ -45,23 +46,43 @@ export class Input extends React.Component<InputProps, undefined> {
       minLength,
       autoFocus,
       inputRef,
+      value,
     } = this.props
 
-    return (
-      <input
-        name={name}
-        type={type}
-        required={required}
-        placeholder={placeholder}
-        className={classNames('Input', className)}
-        onChange={e => this.onEvent(e, onChange, this.props)}
-        onKeyUp={e => this.onEvent(e, onKeyUp, this.props)}
-        onFocus={e => this.onEvent(e, onFocus, this.props)}
-        {...(maxLength ? { maxLength } : {})}
-        {...(minLength ? { minLength } : {})}
-        {...(autoFocus ? { autoFocus: true } : {})}
-        ref={inputRef}
-      />
-    )
+    if (type === 'text-area')
+      return (
+        <textarea
+          value={value && value}
```

Comment:
can we follow working uncontrolled inputs? you can pass the first value with `defaultValue`
What do you think? I leave the reference

with https://reactjs.org/docs/uncontrolled-components.html

Reply:
Ok cool! Thanks